targets/java/source_shared/src/main/java/com/playfab/gsdk/FileLogger.java

```diff
@@ -0,0 +1,107 @@
+package com.playfab.gsdk;
```

Comment:
might wanna add the microsoft header in the files? (or we don't because this is open sourced?